src/components/FormatValues/FormatValues.tsx

```diff
@@ -8,19 +34,210 @@ type Props = {
   onUpdateFormatValue: (formatValue: string | null) => void;
 };
 
+const initialState: FormatterSettings = {
+  prefix: '',
+  suffix: '',
+  precision: '',
+  operation: null,
+  value: '',
+};
+
 const FormatValues: FC<Props> = ({ formatValue, onUpdateFormatValue }) => {
-  const inputRef = useRef(null);
+  const { t } = useTranslation();
+
+  const [dropdown, setDropdown] = useState<'precision' | 'operation'>();
+  const [state, setState] = useState<FormatterSettings>(initialState);
+
+  useEffect(() => {
+    const settings = createFormatterSettings(formatValue);
+    setState({ ...initialState, ...settings });
+  }, [formatValue]);
+
+  const { prefix, suffix, precision, operation, value } = state;
+
+  useDebounce(
+    () => {
+      const settings = serializeFormatterSettings({
+        prefix,
+        suffix,
+        precision,
+        operation,
+        value,
+      });
+      onUpdateFormatValue(settings);
+    },
+    1000,
```

Comment:
I rather suggest to use `200` or `300` ms debounce time - as during testing a one second is providing a delay that can be easily see by the user.